install_pince.sh

```diff
@@ -167,8 +165,15 @@ fi
 
 # shellcheck disable=SC2086
 sudo ${PKG_MGR} ${INSTALL_COMMAND} ${PKG_NAMES} || exit_on_error
+
+# Prepare Python virtual environment
+if [ ! -d ".venv/PINCE" ]; then
+	python3 -m venv .venv/PINCE
+fi
+source .venv/PINCE/bin/activate
+
 # shellcheck disable=SC2086
-sudo ${PIP_COMMAND} install ${PKG_NAMES_PIP} || exit_on_error
+pip3 install ${PKG_NAMES_PIP} || exit_on_error
```

Comment:
On my Arch Linux install, I do have both pip and pip3 and the venv command copies both binaries inside the virtual environment, so there's no need for the env var in my opinion.